src/helpers/commandHelper.ts

```diff
@@ -143,7 +143,8 @@ export class CommandEnvironmentHelper {
 
     args.push('--timeout=' + settings.workspace.editorService.timeout);
     if (vscode.workspace.workspaceFolders !== undefined) {
-      args.push('--local-workspace=' + vscode.workspace.workspaceFolders[0].uri.fsPath);
+      const path = `--local-workspace='${vscode.workspace.workspaceFolders[0].uri.fsPath}'`;
```

Comment:
does the single quotes work on windows?

Reply:
Works on My Machine. Should we use double just in case?